api_v3/services/MediaService.php

```diff
@@ -1227,9 +1227,14 @@ function getVolumeMapAction($entryId)
 
 	private function handleErrorDuringSetResource($entryId, Exception $e)
 	{
+		if ($e->getCode() == KalturaErrors::ENTRY_ID_NOT_FOUND)
+			throw $e; //if no entry found then no need to do anything
+
 		KalturaLog::info("Exception was thrown during setContent on entry [$entryId] with error: " . $e->getMessage());
 		$this->cancelReplaceAction($entryId);
-		if (($e->getCode() == kCoreException::SOURCE_FILE_NOT_FOUND) && (kDataCenterMgr::dcExists(1 - kDataCenterMgr::getCurrentDcId())))
+
+		$errorArr = array(kCoreException::SOURCE_FILE_NOT_FOUND, KalturaErrors::SOURCE_FILE_NOT_FOUND);
```

Comment:
KalturaErrors::SOURCE_FILE_NOT_FOUND is not the error code but has the whole string, only the first string before the ';' is considered the error code